modules/cask-angular-sortable/sortable.js

```diff
@@ -40,7 +40,10 @@ function caskSortableDirective ($log) {
 
       var headers = element.find('th'),
           defaultPredicate,
-          defaultReverse;
+          defaultReverse,
+          noInitialSort;
+
+      noInitialSort = (attrs.noInitialSort === 'true'? true: false);
```

Comment:
The default value would be false which means ```noInitialSort``` is not enabled by default (which is the existing behavior)